### ISSUE TITLE
IOS - Profile - Send validation button is jumping if switching between fields

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#13809]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app
2. Log in with any account
3. Go to Setting - Profile
4. Tap on Phone number
5. Switching between fields

## Expected Result:
Send validation button is not jumping if switching between fields

## Actual Result:
Send validation button is jumping if switching between fields

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [ ] iOS / native

**Version Number:** 1.2.43.1

**Reproducible in staging?:** Yes
 
**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/209248399-5c388078-bdf0-49fe-a2fc-9c7e2edc8ea2.mp4

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by yuwenmemon** [comment]
Was able to reproduce it, but is only going to happen on iPhones with small screens.

**Comment by syedsaroshfarrukhdot** [comment]
@yuwenmemon We need to refactor the AddSecondaryLoginPage.js to use Form.js just like other pages to fix the issue  such as DisplayName.js to fix the issue. AddSecondaryLoginPage.js is not currently using form.js

**Comment by thesahindia** [comment]
It was being refactored but we decided to close the PR because of `#passwordless`. We should close this issue.

**Comment by muhammadbilalarshad** [comment]
Hi,
This issue is occurring due to the Keyboard and how this affects your button's constraints. We need to set the priority of the constraints. I'll explain it further if my proposal is accepted. 

I have 6 years of iOS development experience.

**Comment by mananjadhav** [comment]
Agreed, I thought we were closing all password related issues? Tagging @puneetlath as well.

**Comment by puneetlath** [comment]
I agree we should close this given that the password field will be going away soon. Reopen if you disagree!

**Comment by yuwenmemon** [comment]
Great point, thanks @puneetlath!
